### DISCUSSION TITLE
Getting a 404 error on my page

### DISCUSSION THREAD

**Discussion by quokkaq** [community/community#23525] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve looked for solutions however, I can’t seem to find anything that works for me. I’ve created the repository and double checked to make sure that an index.html file exists with the correct documentation, but when I open my page, I still get the 404 error (index.html file doesn’t seem to be working). Any help would be appreciated! Here’s the link to my repo: <a href="https://github.com/karenLee57/karenLee57.github.io-hangman" rel="nofollow noopener">https://github.com/karenLee57/karenLee57.github.io-hangman</a></p>

**Comment by mpboom** [comment]
<p>I think it’s because your repo is called  <strong>karenLee57.github.io-hangman</strong> instead of  <strong><a href="http://karenLee57.github.io" rel="nofollow noopener">karenLee57.github.io</a></strong>.</p>

**Reply by Bhoovi333** [reply]
Hello @karenlee57...404 error simply means that the file does not exist .We can also consider that the file been removed by publisher.

**Comment by oycyc** [comment]
<p>Hi,</p>
<p>Since your main repository is already <a href="http://name.github.io" rel="nofollow noopener">name.github.io</a>, then the other repositories does not need that in the title. Instead, you only need what you want after the / in your web address. So if you want <a href="http://yourname.github.io/hangman" class="inline-onebox-loading" rel="nofollow noopener">yourname.github.io/hangman</a>, then the repository name has to be “hangman”. Looking at your account, you already created a repository with the name “hangman” with the same code as the repository you linked in this post. I also tried going to the url, and the reason it might not work is because in the “hangman” repository, most likely, you do not have github pages activated. To do this, you simply have to go to the “Settings” tab of your “hangman” repository, and find GitHub Pages. Then make the source to “master” or “gh-pages”, whichever one you used in <a href="http://karenLee7.github.io" rel="nofollow noopener">karenLee7.github.io</a> repository. Then it should work.</p>

**Comment by vaishali-prophecy** [comment]
<p>I have enabled github pages by making source as gh-pages. Still  I am getting 404 error. This github repository is a private repository.</p>

**Comment by tushkyj** [comment]
<p>Check if your master has multiple repositaries…</p>
<p>if multiple repositaries try <a href="http://username.github.io/repositary_name" title="username.github.io/repositary_name" rel="nofollow noopener">username.github.io/repositary_name</a></p>

**Comment by tmetz** [comment, marked as answer]
<p>Hello <a class="mention" href="/karenlee57">@karenlee57</a>!  It does look like <a class="mention" href="/mpboom">@mpboom</a> and <a class="mention" href="/yangciou">@yangciou</a> were on the right track.  If your repository is <code>karenLee57/karenLee57.github.io-hangman</code>, then the URL for your page is <code>https://karenLee57.github.io/karenLee57.github.io-hangman</code>.  That is a mouthful, so I would recommend if you want a project site for this page, to name the repository something shorter.  Your URL for project sites will always be <code>https://karenLee57.github.io/&lt;repository name&gt;</code>.</p>
<p>We frequently see confusion around naming Pages repositories, because you can also have one user site on your account.  For the user site, your URL would just be <code>https://karenLee57.github.io</code>.  However, to activate your user site, you must name your repository in a very exact way.  In this case your user site repository would have to be named <code>karenLee57.github.io</code>.  See below for more information about these two options.</p>
<p>For all those coming to this topic later with a similar issue, below are some common situations where we see <code>404 Not Found</code> errors.</p>
<h2>Check for these common issues:</h2>
<h3>Publishing source and published URL</h3>
<p>There are two kinds of GitHub Pages sites: user/organization sites, and project sites. In most ways the two are identical, but there are differences in how and where they are published.</p>
<p>Every GitHub account is entitled to a single user/organization site, located in a repository called <code>&lt;username&gt;.github.io</code> and accessible at <code>https://&lt;username&gt;.github.io</code> (unless a custom domain is configured). User/organization sites are automatically published from the <code>master</code> branch, and cannot be published from any other source.</p>
<p>Project sites, on the other hand, can be enabled on any repository, and are accessible at <code>https://&lt;username&gt;.github.io/repository-name</code> (again, unless a custom domain is configured). These sites can be published from the <code>master</code> branch, <code>gh-pages</code> branch, or the <code>/docs</code> folder on the <code>master</code> branch, but must be explicitly enabled by configuring a publishing source under your repository’s settings tab.</p>
<h3>Solution: Verify the publishing source and published URL from your repository’s settings tab</h3>
<p>Double check that you are using the correct URL by visiting your repository’s Settings, and scrolling down to the GitHub Pages section.  You should see the URL where your site is published. Here, you can also verify that a publishing source is set, and that it matches your expectations.</p>
<hr>
<h3>No valid default landing page (beware case sensitivity)</h3>
<p>GitHub Pages is designed to serve one of a specific set of files as a default landing page for your site. The preferred files to be served are <code>index.html</code>, <code>index.md</code>, and <code>README.md</code>, in that order. If none of these files exist, your website will show a <code>404 Not Found</code> error, even if there are other HTML files in the repository.</p>
<p>Note that GitHub Pages is case-sensitive, so <code>Index.html</code> (with a capital <code>I</code>) will also fail to function as a default landing page. Most Windows and Mac systems aren’t case-sensitive, but most web servers are, including GitHub Pages. This means that while your personal computer might consider <code>Index.html</code> and <code>index.html</code> to refer to the same file, GitHub Pages won’t. Ensuring the use of entirely lowercase filenames is the best way to avoid these issues.</p>
<h3>Solution: Create one of these files</h3>
<p>Create a file named <code>index.html</code>, <code>index.md</code>, or <code>README.md</code> to act as your default document (home page).  You can also rename an existing HTML or Markdown file to one of these 3 filenames.</p>
<hr>
<h3>Repository is a private repo on a free plan</h3>
<p>GitHub Pages in private repositories is only available on the following plans: GitHub Pro, GitHub Team, GitHub Enterprise Cloud, and GitHub Enterprise Server.</p>
<p>Sometimes people switch their repository’s visibility to private.  This will unpublish the Pages site, and switching back to public will not automatically re-publish.</p>
<h3>Solution: Verify plan</h3>
<p>Double check that your plan supports Pages in private repositories.  If it does not, make sure the repository is public, and then push a commit to build the site again.</p>
<hr>
<h3>I just published, but am still getting a 404 error</h3>
<p>It’s possible that you just published the project or published a new commit but aren’t seeing any new updates.</p>
<h3>Solution: Allow a few minutes</h3>
<p>It can take up to 10 minutes for GitHub Pages to build and publish your site. Allow some time for the build to complete, and also clear your browser’s cache.</p>
<hr>
<h2>Additional Resources:</h2>
<p>If you are here looking for a solution to your <code>404 Not Found</code> issue, I hope we’ve been able to help with the information above!  Here are some additional resources that may be helpful:</p>
<p>About GitHub Pages:<br>
<a href="https://help.github.com/en/github/working-with-github-pages/about-github-pages" class="onebox" target="_blank">https://help.github.com/en/github/working-with-github-pages/about-github-pages</a></p>
<p>Configuring a publishing source for your GitHub Pages site:<br>
<a href="https://help.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site" class="onebox" target="_blank">https://help.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site</a></p>

**Comment by aryajain1290** [comment]
<p>hi,</p>
<p>i am getting an error while hosting my repository on git hub pages.  i have followed all the steps properly while hosting on git hub pages but still the error is coming. I have used the initialized readme option.<br>
please reply quickly</p>

**Comment by JavascriptLearner815** [comment]
<p>I have another problem like this. When I use links, my 404 page shows up. Link: <a href="https://javascriptlearner815.github.io/" rel="noopener nofollow ugc">https://javascriptlearner815.github.io/</a></p>

**Comment by JavascriptLearner815** [comment]
<p>Name change. That’s why.</p>

**Comment by sudhanshuranjanAki** [comment]
<p>We have expertise to solve the problem that you are asking why don’t you hire us to solve your problem we are <a href="https://srchout.com/website-development-company" rel="noopener nofollow ugc">website development company</a></p>

**Comment by Cattail411** [comment]
<p>Can anyone smarter than me help me with this? This link doesn’t work and I don’t know why.</p>
<p><a href="https://cattail411.github.io/WDD130/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://cattail411.github.io/WDD130/</a></p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="Cattail411" data-post="11" data-topic="10405">
<div class="title">
<div class="quote-controls"></div>
 Cattail411:</div>
<blockquote>
<p><a href="https://cattail411.github.io/WDD130/">https://cattail411.github.io/WDD130/</a></p>
</blockquote>
</aside>
<p>it has a folder that contains the default <code>index.html</code>, so you need to include that at the URL too</p>
<p><a href="https://cattail411.github.io/WDD130/wdd130/">https://cattail411.github.io/WDD130/wdd130/</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/b/eb65a6be57a21aa477e859c3b50bce826c59861b.png" data-download-href="/uploads/short-url/xApZxmII8CKaMiksVRU1vftR523.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077559-c700fbd3-eed6-4f8a-a5a9-0fc5f613fa28.png" alt="image" data-base62-sha1="xApZxmII8CKaMiksVRU1vftR523" width="690" height="178" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/b/eb65a6be57a21aa477e859c3b50bce826c59861b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">773×200 10.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Reply by Tmskinner01** [reply]
Thanks so much, I was having the same issue and this helped a lot

**Comment by dipali-31** [comment]
https://github.com/dipali-31/dipali-31.github.io.git my https it is showing a error of 404

**Comment by bhaskar8790** [comment]
![Screenshot 2024-04-30 111143](https://github.com/community/community/assets/166101818/c0535536-3a7e-4711-9697-c83807610c11)
why its showing like this after deploying complete and when click on that link and showing in a new tab

**Comment by RedaChahid** [comment]
https://github.com/RedaChahid/
i have an issue 404 this is not the web page you are looking for
